app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] According to the project requirement, kindly change user to author hence the foreign keys would be author_id, you can change this from the migration file so this line should look like this `belongs_to :author, class_name: 'User'`